cmd/kubeadm/app/cmd/join.go

```diff
@@ -308,44 +309,55 @@ func newJoinOptions() *joinOptions {
 // newJoinData returns a new joinData struct to be used for the execution of the kubeadm join workflow.
 // This func takes care of validating joinOptions passed to the command, and then it converts
 // options into the internal JoinConfiguration type that is used as input all the phases in the kubeadm join workflow
-func newJoinData(cmd *cobra.Command, args []string, options *joinOptions, out io.Writer) (*joinData, error) {
+func newJoinData(cmd *cobra.Command, args []string, opt *joinOptions, out io.Writer) (*joinData, error) {
```

Comment:
Note for reviewers: 
I was forced to rename `options` to `opt` because it was hiding an imported package, and this, unfortunately, adds some noise into the PR.
The relevant changes on `newJoinData ` are at L349 and L379

Reply:
ok, got it.
i did add a comment about that:
`the opt rename seems out of scope, but this is fine.`

will delete it.